nixos/tests/common/acme/client/default.nix

```diff
@@ -9,6 +9,11 @@ in {
     defaults = {
       server = "https://${caDomain}/dir";
       email = "hostmaster@example.test";
+      # Avoid a random 0-8 minute sleep when testing renewals.
+      # We are not using LE servers in testing so this is not
+      # going to impact their load.
+      # See https://github.com/go-acme/lego/issues/1656
+      extraLegoRenewFlags = ["-no-random-sleep"];
```

Comment:
Considering this is removed in the subsequent commit, would it make sense to just not add it in this commit?

Reply:
Hm, I'm normally pretty adverse to editing commits because it alerts all the related issues again when you force push. Also I think these commits show a good evolution of the feature?